.circleci/config.yml

```diff
@@ -52,6 +58,32 @@ jobs:
             - build/libs
             - build.gradle
             - app_version.txt
+  build_docker:
+    executor: deployer
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/app
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Build container and publish to docker hub
+          command: |
+            APP_VERSION=$(cat ./app_version.txt)
+            docker build -f docker/Dockerfile.aws --rm=false . \
+               --tag "mojdigitalstudio/deliusapi:$APP_VERSION" \
```

Comment:
`deliusapi` is the existing misnamed repo so using that for now

Reply:
Yes I have privs to create a new one